packages/authentication-oauth/src/express.ts

```diff
@@ -48,11 +46,7 @@ export default (options: OauthSetupSettings) => {
       req.session.redirect = redirect;
       req.session.query = query;
 
-      res.redirect(`${path}/connect/${name}?${qs.stringify(query as any)}`);
-    });
-
-    authApp.get('/:name/callback', (req: any, res: any) => {
-      res.redirect(`${path}/connect/${req.params.name}/callback?${qs.stringify(req.query)}`);
```

Comment:
The new grant version mounts at `/prefix` instead of `/prefix/connect` - so redirecting is not needed anymore, but the request can just be passed on to the next handler.